sdk/spring/azure-spring-boot-samples/README.md

```diff
@@ -0,0 +1,50 @@
+# Azure Spring Sample shared client library for Java
+
+## Environment setup
+
+### Prerequisites
+
+- [Java Development Kit (JDK)][jdk_link] with version 8 or above
+- [Maven][maven] 3.0 and above
+- An Internet connection
+- A Windows machine (necessary if you want to run the app on Windows)
+- An OS X machine (necessary if you want to run the app on Mac)
+- A Linux machine (necessary if you want to run the app on Linux)
+- Git
+- Microsoft Azure subscription
+    - You can create a free account at: https://azure.microsoft.com
+
+### Clone source
+There are two scenarios for demonstrating.
+
+#### Clone from master branch
+It means run the master branch, all the code, dependencies, and readme files are up to date.
+Execute `git clone https://github.com/Azure/azure-sdk-for-java.git`.
+
+#### Clone source from specific version tag
+It means run the special release branch, all the code, dependencies, and readme files are versions of the corresponding release version.
+Find out which release tag you want to use, then replace tag name and execute `git clone -b <replace-the-tag-you-want-to-use> https://github.com/Azure/azure-sdk-for-java.git`.
+After cloning the code, you may be able to skip step **Compile the project** if all dependencies can be found.
+
+>**Note:** 
+There's a [link][azure_spring_release_tags] to query the `azure-spring` release tag, if there are too many returns, you can modify the query keyword value to accurately query. 
+
+### Compile the project
+1. Traverse to the root directory
+1. Build the whole product by executing the following command which may take several minutes:
+    * `mvn clean install -Dmaven.javadoc.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true -Drevapi.skip=true -Djacoco.skip=true​ -DskipTests -Dparallel-test-playback`
+
+After executing the above steps, you can build the spring project only for the developing purpose:
+1. Traverse to spring directory:
```

Comment:
We could skip this step since all spring projects are built from the last command.

Reply:
Changed to:
After cloning the code, you may be able to skip step **Compile the project** if all dependencies can be found.

Reply:
But how to define all dependencies can be found? After users compile the sdk/spring?

Reply:
The latest release doesn't include the aad-starter, it's better to update the maven dependencies first after cloning the special release tag, the missing dependencies will be found, let me update the description.